remote-query/remote-query-server/src/main/java/org/infinispan/query/remote/impl/logging/Log.java

```diff
@@ -66,4 +66,9 @@
    @Message(id = 28016, value = "Query performed in a cache ('%s') that has an unknown format configuration. " +
          "Please configure the cache encoding as 'application/x-protostream' or 'application/x-java-object'")
    void warnNoMediaType(String cacheName);
+
+   //TODO [anistor] This warning will become ERROR in version 12
```

Comment:
Could you add a line in upgrading.adoc about this?

Reply:
done